packages/documentation-site/patternfly-docs/content/design-guidelines/styles/icons/icons.css

```diff
@@ -1,8 +1,8 @@
-.pf-v5-c-divider.ws-icons-divider {
```

Comment:
This stylesheet is another good one to review as a whole and see if we can't use some existing components/layouts/styles. With the styles here, one big one that sticks out is we're not using spacer vars for spacers we support.

Reply:
Made a followup here: https://github.com/patternfly/patternfly-org/issues/4020